### ISSUE TITLE
Fix link to CONTRIBUTING.md file from README.md

### ISSUE THREAD

**Issue by osg** [dita-ot/docs#267]
Actual: [5]: `CONTRIBUTING.md`
Expected: [5]: `.github/CONTRIBUTING.md`

Notes: Tried to make a PR via the GUI, could not sign off. Tried via the command line, and I do not have permissions to create a new branch and create a PR. Therefore, I am creating this issue. Maybe I could read the `CONTRIBUTING.md` before fixing its link. heh

```
1220  git pull
1221  git checkout -b fix-link-in-readme
1222  ls
1223  vi README.md 
1224  git status
1225  git add README.md 
1226  git commit
1227  git push
1228  git pull
1229  git branch --set-upstream-to=origin/fix-link-in-readme fix-link-in-readme
```
I see:
```
$ git push
ERROR: Permission to dita-ot/docs.git denied to osg.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
Probably a user error. Will need to come back to this on a fresh brain.

**Comment by osg** [comment]
I used SSH to clone the repo if that makes a difference.

**Comment by infotexture** [comment]
@osg Good catch, that link is indeed broken. 🙏 

You'll need to [fork](https://help.github.com/articles/fork-a-repo/) the repository first in order to push your changes. 

- For command line instructions, see [submitting pull requests](https://github.com/dita-ot/docs/blob/develop/.github/CONTRIBUTING.md#submitting-pull-requests). 
- To edit the file in your browser, click the pencil icon in [README file](https://github.com/dita-ot/docs/blob/develop/README.md) header in the GitHub UI, which should create the fork automatically.

**Comment by infotexture** [comment]
Resolved with #268.
